assets/quick-order-list.js

```diff
@@ -116,7 +116,7 @@ if (!customElements.get('quick-order-list')) {
         this.cartUpdateUnsubscriber = subscribe(PUB_SUB_EVENTS.cartUpdate, (event) => {
           const variantIds = [];
           this.querySelectorAll('.variant-item').forEach((item) => {
-            variantIds.push(item.dataset.variantId);
+            variantIds.push(parseInt(item.dataset.variantId));
```

Comment:
We need to update line 124 too: `(event.cartData.items && variantIds.some((element) => !event.cartData.items.includes(element)))`

`items` is an array of objects but we're checking for presence of a variantId, so `.includes` will always return false.

Reply:
Good catch 👍